src/components/Metrics/TrafficDetails.tsx

```diff
@@ -58,7 +60,7 @@ class TrafficDetails extends React.Component<TrafficDetailsProps, TrafficDetails
   }
 
   render() {
-    const rateIntervalName = getName(this.props.rateInterval).toLowerCase();
+    const rateIntervalName = getName(this.props.duration).toLowerCase();
```

Comment:
To be consistent, I think this should be durationName or maybe durationLabel.

Reply:
Ouch. I missed that one. Fixed!